webpack/scenes/Subscriptions/components/SubscriptionsTable/SubscriptionsTable.js

```
@@ -78,40 +77,33 @@ class SubscriptionsTable extends Component {
     const allSubscriptionResults = this.props.subscriptions.results;
 
     const checkAllRowsSelected = () =>
-      allSubscriptionResults.length === this.state.selectedRows.length;
+      allSubscriptionResults.length === this.props.selectedRows.length;
 
     const updateDeleteButton = () => {
-      this.props.toggleDeleteButton(this.state.selectedRows.length > 0);
+      this.props.toggleDeleteButton(this.props.selectedRows.length > 0);
     };
     return ({
       allRowsSelected: () => checkAllRowsSelected(),
       selectAllRows: () => {
         if (checkAllRowsSelected()) {
-          this.setState(
-            { selectedRows: [] },
-            updateDeleteButton,
-          );
+          this.props.onSelectedRowsChange([]);
+          this.setState(updateDeleteButton);
         } else {
-          this.setState(
-            { selectedRows: allSubscriptionResults.map(row => row.id) },
-            updateDeleteButton,
-          );
+          this.props.onSelectedRowsChange(allSubscriptionResults.map(row => row.id));
+          this.setState(updateDeleteButton);
```

Same comment for this one, except pass `true`